cmake/FindMKL.cmake

```diff
@@ -66,6 +66,8 @@ if(UNIX)
   list(APPEND MKL_LINK_PREFIX "-L${MKL_LIB_DIR}")
   set(LIB_PREFIX "-l")
   set(OPENCL_LIBNAME "OpenCL")
+  set(SYCL_LIB_PREFIX "-f")
+  set(SYCL_LIBNAME "sycl")
```

Comment:
The `-fsycl` flag isn't specifying a library, but instead telling the linker that there are SYCL source files. So I think it would be more accurate to replace these two lines with e.g.

```
  set(SYCL_LINK_FLAGS "-fsycl")
```

and use `SYCL_LINK_FLAGS` below.

Reply:
Done